Makefile

```diff
@@ -1,78 +1,86 @@
 MAKEFLAGS += --no-builtin-rules
 MAKEFLAGS += --no-builtin-variables
 
-# allow overriding the name of the venv directory
-VENV_DIR ?= venv
```

Comment:
why is this elided? don't the make rules work with a `VENV_DIR` variable?

Reply:
I believe `uv` sets the venv directory with `UV_PROJECT_ENVIRONMENT`
https://docs.astral.sh/uv/configuration/environment/#uv_project_environment

Reply:
Unfortunately `uv` doesn't support custom virtual environment names currently (https://github.com/astral-sh/uv/issues/1422), so this variable cannot be used for anything and so should be elided to avoid confusion

Reply:
I think this was resolved in what @alexarice via https://github.com/astral-sh/uv/pull/6834, so it would be nice to hook it up to what we currently have been using

Reply:
Good spot! I've updated the Makefile to use this to respect it as previously